aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/apigateway/LambdaRequestEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.amazonaws.services.lambda.runtime.events.apigateway;
+
+import com.amazonaws.services.lambda.runtime.events.apigateway.RequestSource;
+
+/**
+ * Interface to abstract APIGWY and ALB request classes, reduce complexity and increase efficiency.
+ */
+public interface LambdaRequestEvent {
```

Comment:
Can this be renamed to something like `HttpRequestEvent`?

Reply:
Done.